howdoi/howdoi.py

```diff
@@ -13,7 +13,6 @@
 import os
 import random
 import re
-import requests
 import requests_cache
```

Comment:
We use requests_cache to speed up query times. Is there a similar library for urllib2?